locale/messages.pot

```diff
@@ -8077,6 +8191,10 @@ msgstr ""
 msgid "SUICIDE_BUTTON_TOOLTIP"
 msgstr ""
 
+#: ../src/gui_common/menus/PlanetSettings.tscn:437
+msgid "SULFIDE_LEVELS"
```

Comment:
We don't have sulfide in the game so I think for clarity this translation key should be "HYDROGEN_SULFIDE_..."